auter.yumdnfModule

```diff
@@ -16,6 +16,11 @@ function prepare_updates() {
 
       # If check-update has an exit code of 100, updates are available.
       if [[ "${RC}" -eq 100 ]]; then
+        # Run any pre-prep scripts
```

Comment:
This behaviour is a little confusing. Here pre-prep scripts will only run if there are updates available AND predownloadupdates == yes. post-prep will run regardless of either of the above. I think the right behaviour might be to always run pre/post-prep regardless.